DOCS/man/options.rst

```diff
@@ -1846,10 +1846,11 @@ Video
 Audio
 -----
 
-``--audio-pitch-correction=<yes|no>``
-    If this is enabled (default), playing with a speed different from normal
-    automatically inserts the ``scaletempo2`` audio filter. For details, see
-    audio filter section.
+``--audio-pitch-correction=<no|rubberband|scaletempo2|yes>``
```

Comment:
Shouldn't `scaletempo` be a choice too?

Reply:
I don't think it offers anything of value here, it's just scaletempo2 but worse. I believe @jeeb intended to remove it even, but it hasn't happened yet so maybe he can comment on it.

Reply:
People did complain about scaletempo2 before (https://github.com/mpv-player/mpv/issues/8705), but maybe https://github.com/mpv-player/mpv/pull/12052 fixed it? I don't know; I don't really change speed much except if I'm practicing a song and it's not like I'm paying attention to how the audio sounds like then.

Reply:
Well the claim was that for small speed changes close to 1, scaletempo was better than scaletempo2. That might actually be fixed now though.

Reply:
I don't think that PR made a difference for that.

Reply:
Pointless to bikeshed, so just added scaletempo and improved docs. This can be removed in the future when we do remove scaletempo.